integration-tests/create-react-app/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "create-react-app",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "react": "^15.6.1",
+    "react-dom": "^15.6.1"
+  },
+  "optionalDependencies": {
+    "react-scripts": "1.0.11"
+  },
+  "devDependencies": {
+    "@percy-io/react-percy": "^0.1.5",
+    "webpack": "^3.5.5"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test --env=jsdom",
+    "eject": "react-scripts eject",
+    "percy": "react-percy"
```

Comment:
With this change, `react-percy` should work pretty seamlessly with `create-react-app` out of the box. Add it as a dependency, add a percy script to the package, and start writing snapshots.